install.sh

```diff
@@ -1,15 +1,27 @@
 #!/usr/bin/env bash
+
 # Install the CircleCI CLI tool.
-# https://github.com/circleci-public/circleci-cli
+# https://github.com/CircleCI-Public/circleci-cli
+#
+# Dependancies: curl, cut
+#
+# The version to install and the binary location can be passed in via VERSION and DESTDIR respectively.
+#
 
 set -o errexit
-set -o nounset
 
-echo Installing CircleCI CLI
+echo "Starting installation."
 
-RELEASE_URL="https://api.github.com/repos/CircleCI-Public/circleci-cli/releases/latest"
+# GitHub's URL for the latest release, will redirect.
+LATEST_URL="https://github.com/CircleCI-Public/circleci-cli/releases/latest/"
 DESTDIR="${DESTDIR:-/usr/local/bin}"
 
+if [ -z "$VERSION" ]; then
+	VERSION=$(curl -sLI -o /dev/null -w '%{url_effective}' $LATEST_URL | cut -d "v" -f 2)
```

Comment:
What is `'%{url_effective}'` for?

Reply:
The URL GitHub provides is just a redirect. This captures that new URL.